driver/node.go

```diff
@@ -116,11 +131,18 @@ func (d *Driver) NodeUnstageVolume(ctx context.Context, req *csi.NodeUnstageVolu
 	})
 	ll.Info("node unstage volume called")
 
-	err := d.mounter.Unmount(req.StagingTargetPath)
+	mounted, err := d.mounter.IsMounted("", req.StagingTargetPath)
 	if err != nil {
 		return nil, err
 	}
 
+	if mounted {
+		err := d.mounter.Unmount(req.StagingTargetPath)
+		if err != nil {
+			return nil, err
+		}
+	}
```

Comment:
For consistency in logging, should we also log that the device is already not mounted here like for the other actions?

Reply:
You're right. I've missed this part. Fixed and force pushed it 👍